4.23/ExampleProject/Plugins/PlayFab/Source/PlayFabCpp/Private/Core/PlayFabAuthenticationInstanceAPI.cpp

```diff
@@ -116,7 +115,7 @@ void UPlayFabAuthenticationInstanceAPI::OnGetEntityTokenResult(FHttpRequestPtr H
     if (PlayFabRequestHandler::DecodeRequest(HttpRequest, HttpResponse, bSucceeded, outResult, errorResult))
     {
         if (outResult.EntityToken.Len() > 0)
-            this->GetOrCreateAuthenticationContext()->SetEntityToken(outResult.EntityToken);
+            // context->SetEntityToken(outResult.EntityToken);
```

Comment:
This may be odd. How will this compile? Will the line below count as a part of the if check? Or does this if check for something and then ... ?